bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/dto/ButtonModel.java

```diff
@@ -23,12 +25,46 @@
  * @author Christoph Weitkamp - Initial contribution
  */
 @XmlAccessorType(XmlAccessType.FIELD)
-@XmlType(propOrder = { "lastpressedtimestamp" })
+@XmlType(propOrder = { "name", "lastpressedtimestamp" })
```

Comment:
First of all no, because `propOrder` specifies the order for XML Schema elements when class is mapped to a XML Schema and "id" and "identifier" are attributes.

But that makes me wonder why I need it at all. I do not write XML. I will try to remove it.

Reply:
Gah, you are right. It has been too long since I used the xml annotations. 😄